week-3/Homework/mandatory/1-oxygen-levels.js

```diff
@@ -27,32 +26,37 @@ console.log(newdessert); // returns ice cream
 /* + + + + + + + */
 /* Now try the exercise */
 
-function safeLevels() {
-
+function safeLevels(num) {
+  let percent = num.map((p) => p.replace("%", ""));
+  let found = percent.find(function (item) {
+    return item > 19.5 && item < 23.5;
+  });
+  return found + "%";
 }
```

Comment:
The function is correct but you haven't turned the string of the map method into number. It needs to be changed to number inside the find() mehod.